tests/cli/test_cmds_mark_for_deployment.py

```diff
@@ -207,7 +207,7 @@ def test_slack_deploy_notifier(mock_get_authors, mock_client):
     assert sdn.notify_after_good_deploy() is None
     assert sdn.notify_after_auto_rollback() is None
     assert sdn.notify_after_abort() is None
-    assert fake_psc.post.call_count >= 0, fake_psc.post.call_args
```

Comment:
lol, so this assertion could never fail before?

Reply:
Apparently lol